Wikipedia/Code/ErrorWorkaround.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ErrorWorkaround.swift
```

Comment:
I'll move the comments i wrote in `WMFImageController` here, but it's basically something weird going on with casting swift `ErrorType` objects to `NSError`.  this is only extending `WMFImageController` because it's "ground zero" for this issue (a Swift class with custom errors that we're referencing in ObjC).